projects/storefrontapp-e2e-cypress/cypress/helpers/accessibility/tabbing-order/order-history.ts

```diff
@@ -0,0 +1,46 @@
+import { checkAllElements, TabElement } from '../tabbing-order';
+import { doPlaceOrder } from '../../order-history';
+import { TabbingOrderTypes } from '../tabbing-order.config';
+
+export function orderHistoryNoOrdersTabbingOrder(config: TabElement[]) {
+  cy.visit('/my-account/orders');
+  cy.get('cx-order-history a')
+    .first()
+    .focus();
+  checkAllElements(config);
+}
+
+export function orderHistoryWithOrdersTabbingOrder() {
+  doPlaceOrder().then((orderData: any) => {
+    const config: TabElement[] = [
```

Comment:
please create 2 entries in our config.
- orderHistoryNoOrders
- orderHistoryOrders

Reply:
Config for orderHistoryOrders needs to be dynamically generated based on the result from `doPlaceOrder()` and can't just be placed in our config.